### DISCUSSION TITLE
[in very long list] Неудобное отображение пометок отбора в списке уникальных значений колонки

### DISCUSSION THREAD

**Discussion by tormozit** [totumonline/totum-ru-issues-and-discussions#193] [Есть идея, а что если...]
Пометки накладываются на текст. Ожидал, что текст не будет пересекаться с пометками.

![изображение](https://user-images.githubusercontent.com/11896251/184499110-fbbb1280-9e17-4ca1-95f7-0a51b4bf30d6.png)

**Comment by ttm-alexeypolunin** [comment]
Запишем в лист пожеланий :)
